data/the-lazy-learner.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Sudhansu",
+  "url": "https://github.com/the-lazy-learner",
+  "image": "https://github.com/the-lazy-learner.png",
+  "why": "To collaborate and learner better ways of life",
```

Comment:
```suggestion
  "why": "To collaborate and learn better ways of life",
```

Reply:
Grammatical error

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Thanks for pointing out